Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -257,6 +256,66 @@ function GetMailbox {
     }
 }
 
+<#
+.SYNOPSIS
+Checks if a set of Calendar Logs is from the Organizer.
+#>
+function SetIsOrganizer {
+    param(
+        $CalLogs
+    )
+    $IsOrganizer = $false
+
+    foreach ($CalLog in $CalLogs) {
+        if ($CalendarItemTypes.($CalLog.ItemClass) -eq "IpmAppointment" -and
+            $CalLog.ExternalSharingMasterId -eq "NotFound" -and
+            ($CalLog.ResponseType -eq "1" -or $CalLogs.ResponseType -eq "Organizer")) {
+            $IsOrganizer = "True"
+            Write-Verbose "IsOrganizer: [$IsOrganizer]"
+            return $IsOrganizer
+        }
+    }
+    Write-Verbose "IsOrganizer: [$IsOrganizer]"
+    return $IsOrganizer
+}
+
+function SetIsRoom {
+    param(
+        $CalLogs
+    )
+    $IsRoom = $false
+    # Simple logic is if RBA is running on the MB, it is a Room MB, otherwise it is not.
+    foreach ($CalLog in $CalLogs) {
+        if ($CalendarItemTypes.($CalLog.ItemClass) -eq "IpmAppointment" -and
+            $CalLog.ExternalSharingMasterId -eq "NotFound" -and
+            $CalLog.Client -eq "ResourceBookingAssistant" ) {
+            $IsRoom = "True"
+            return $IsRoom
+        }
+    }
+    return $IsRoom
+}
+
+function SetIsRecurring {
+    param(
+        $CalLogs
+    )
+    Write-Host -ForegroundColor Yellow "Looking for signs of a recurring meeting."
+    $IsRecurring = $false
+    # See if this is a recurring meeting
+    foreach ($CalLog in $CalLogs) {
+        if ($CalendarItemTypes.($CalLog.ItemClass) -eq "IpmAppointment" -and
+            $CalLog.ExternalSharingMasterId -eq "NotFound" -and
+            $CalLog.CalendarItemType.ToString() -eq "RecurringMaster" ) {
+            $IsRecurring = "True"
```

Comment:
Same thing here. Let's be consistent, either a string or a boolean

Reply:
Done